ember-app/app/models/new/board.js

```diff
@@ -92,6 +92,16 @@ var Board = Model.extend({
       return combined;
     }
   }),
+  issuesById: function(){
+    return _.groupBy(this.get('issues'), (issue)=> {
+      return issue.get('id');
+    });
+  }.property('issues.[]'),
+  issuesByRepo: function(){
+    return _.groupBy(this.get('issues'), (issue)=> {
```

Comment:
It seems like we don't really need the actual issues per repo, just the unique list of `repo.full_name`? If I'm reading this correctly, we can use `countBy` to save on allocation of new arrays to hold the repos' issues.